eclass/qt5-build.eclass

```diff
@@ -193,10 +193,6 @@ qt5-build_src_prepare() {
 		sed -i -e "/^QMAKE_CONF_COMPILER=/ s:=.*:=\"$(tc-getCXX)\":" \
 			configure || die "sed failed (QMAKE_CONF_COMPILER)"
 
-		# Respect toolchain and flags in config.tests
-		find config.tests/unix -name '*.test' -type f -execdir \
```

Comment:
The upstream commit dropping the last references to -nocache is only 5.8 and later - should we make this conditional, or is it fine to nuke completely?

Reply:
yep, should be made conditional.